canonical-web-and-design/ubuntu.com
# Get subscriptions data

## Done

- Add subscription data to the window.
- Fetch data from the window and insert into the react query store.

## QA

- [Turn on dev mode](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/HACKING.md#javascript).
- Visit /advantage?test_backend=true.
- Click on the devtools.
- You should see some data that has been loaded.


## Issue / Card

Fixes canonical-web-and-design/commercial-squad#100.
